backend/package.json

```diff
@@ -20,8 +20,8 @@
   ],
   "main": "index.ts",
   "scripts": {
-    "ng": "./node_modules/@angular/cli/bin/ng",
-    "tsc": "./node_modules/typescript/bin/tsc",
+    "ng": "node ./node_modules/@angular/cli/bin/ng",
+    "tsc": "node ./node_modules/typescript/bin/tsc",
```

Comment:
This looks committed by mistake - obviously not related to price data :)

Reply:
It isn't, maybe I should separate the PR to avoid confusion, but this is what enables building for Windows environment